raiden_libs/test/fixtures/web3.py

```diff
@@ -0,0 +1,157 @@
+import logging
+import pytest
+import gevent
+
+import rlp
+from eth_utils import decode_hex, denoms
+from ethereum.transactions import Transaction
+from eth_tester import EthereumTester, PyEVMBackend
+
+
+from web3 import Web3
+from web3.providers.eth_tester import EthereumTesterProvider
+from web3.providers.rpc import HTTPProvider
+
+from raiden_libs.utils import (
+    address_from_signature,
+    keccak256,
+)
+# import microraiden.utils.contract
+
+DEFAULT_TIMEOUT = 5
+DEFAULT_RETRY_INTERVAL = 3
+FAUCET_ALLOWANCE = 100 * denoms.ether
+INITIAL_TOKEN_SUPPLY = 200000000000
+WEB3_PROVIDER_DEFAULT = 'http://localhost:8545'
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='session')
+def ethereum_tester():
+    """Return instance of an Ethereum tester"""
```

Comment:
*Returns an instance...* 

There are other instances of *return* vs. *returns*